### ISSUE TITLE
[Bug] My Manga Reader CMS (Comic Space): ERR_CONNECTION_TIMED_OUT

### ISSUE THREAD

**Issue by mariodac** [tachiyomiorg/tachiyomi-extensions#4975]
## Device information
- Tachiyomi version:  10.5
- Android version: 7.1.2

## Source information
- Name: My Manga Reader CMS (Comic Space)
- Extension version: 1.2.52

## Steps to reproduce
1. Open the source Comic Space
2. Search something, for example "a bad friend"
3. Open it and try to read a chapter

### Expected behavior
After step 1 see popular manga, after step 3 see loaded pages

### Actual behavior
The manga list is empty even if you search for something, when searching showing the repeat button, select open in webview and receive the error ERR_CONNECTION_TIMED_OUT

## Other details
Searching the internet, discovering that they changed their domain, in the app is comicspace.com.br, but currently they have domain acomic.space

**Comment by scb261** [comment]
The site was mistakenly removed #5089 instead of changing url. And I'm not sure if it's still has the same theme

**Comment by alessandrojean** [comment]
The site is dead for a long period of time, and the domain shows an Apache screen.

If they do change something in the future, and it starts working again, request it by opening a new issue.
